test/e2e/fixture/zomes/test/Cargo.toml

```diff
@@ -9,6 +9,6 @@ name = "test_wasm"
 crate-type = [ "cdylib", "rlib" ]
 
 [dependencies]
-hdk3 = { git = "https://github.com/holochain/holochain", rev = "63b9fca72bf88b5760ad106256d3882f143d4f25", package = "hdk3" }
-serde = "1.0.104"
+hdk = { version = "0.0.100-alpha.1", git = "https://github.com/holochain/holochain", rev = "5812b41398e857134f780621dc1aa7bc6c05c5f0", package = "hdk" }
```

Comment:
Is there a reason to add this version specifier in additon to the git ones? Looking at [the cargo documentation](https://doc.rust-lang.org/nightly/cargo/reference/specifying-dependencies.html#multiple-locations) it looks like this causes the version number only to be used when publishing the crate, but this crate will never be published.

Reply:
I had to do this to get it to work when compiling the dna over on holochain-conductor-api, but I'm going to try just using the crate, which is supposed to work now.

Reply:
Now that the HDK version isn't tied to the desired holochain version for `install-holochain.sh`, there is nothing that will bust the GitHub actions cache when the holochain version is changed. I think it's important to make the following change to the GitHub actions file:

```diff
    - uses: actions/cache@v2
      with:
        path: |
          ~/.cargo/registry
          ~/.cargo/git
          target
-         key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}
+         key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}-${{ hashFiles('ci_scripts/install_holochain.sh`) }}
```

Reply:
done

Reply:
Did you forget to push?

Reply:
Yep!